router/server.go

```diff
@@ -20,6 +20,9 @@ type Router struct {
 func NewRouter(ctx context.Context, h models.HandlerInterface, port int) *Router {
 	gMux := mux.NewRouter()
 
+	gMux.PathPrefix("/plugins/").
```

Comment:
@kalradev will you update the Meshery Extensibility doc with this proposed path? It'll be easier to review in context of other extension points in the design spec.

Reply:
Sure, After completing this will do

Reply:
Sounds real good.